docs/guide/personalization/developer_guide/recommendation_api.md

```diff
@@ -189,49 +194,56 @@ jsonpCallback({
+|Response|`Expires`|Gives the date/time after which the response is outdated|`Expires: Thu, 01 Dec 2013 16:00:00 GMT`|-|
 
-The last modification timestamp indicates a change, that could influence the recommendation response. It depends on an updated recommendation calculation, an update of an item or some scenario configuration changes. The expiration timestamp is a best-effort prediction based on the model building configuration and provided context. The shortest expiration period is 5 minutes from the request time. The longest is 24 hours. In the table below several examples are illustrated:
+The last modification timestamp indicates a change that could influence the recommendation response. 
+It depends on an updated recommendation calculation, an update of an item or certain scenario configuration changes. 
 
-| Model | Context | Expiration time | Description |
-|------|------|-----|-----|
-| Bestselling products last 7 days | no context | 24 hours | The model with the 7 days scope is usually built once a day. It can be easily cached for 24 hours. It has no context and can therefore be cached globally for all the users of a customer. |
-| Also bought products in the last month | current product | 24 hours | The model with the 30 days scope is usually built once a day. The context is always the same. It can be cached for every product. The same result can be used for all users of a customer. |
-| Also consumed read articles in the last hour | current article | 30 minutes | Models with a short scope are usually built several times a day or even per hour. In this case the expiration time is set to the half of the model build frequency/period. |
-| Personalized recommendation based on the user's statistic | customers click history | now | Although the statistic model is not updated within minutes, it is very likely that the context will be changed shortly (customer clicks another product and therefore the click is added to his history). The expiration time should not be much longer than the user activity on the web page. |
+The expiration timestamp is a best-effort prediction based on the model configuration and provided context. 
+The shortest expiration period is 5 minutes from the request time, the longest is 24 hours. 
 
-In most cases you do not need to calculate the expiration time manually. The table above is provided for the orientation, how the Expires header of the HTTP response is calculated by the recommendation engine and already provided to your caching system. You just need to make sure that the Expires header is used in the configuration of your caching system instead of a static value out of your configuration.
+In most cases you do not have to calculate the expiration time manually. 
+You just need to make sure that the `Expires` header is used in the configuration of your caching system instead of a static value out of your configuration.
```

Comment:
```suggestion
You just need to make sure that the `Expires` header is used in the configuration of your caching system
instead of a static value from your configuration.
```